docker-compose-dev.yml

```diff
@@ -0,0 +1,318 @@
+# an ephemeral instance of ChRIS backend services for local development
+#
+# warning: /var/run/docker.sock is mounted into some services (notably pman)
+
+services:
+  chrisomatic:
+    image: ghcr.io/fnndsc/chrisomatic:0.8.2
+    profiles:
+      - tools
+    volumes:
+      - "./chrisomatic/chrisomatic.yml:/chrisomatic.yml:ro"
+      - "/var/run/docker.sock:/var/run/docker.sock:rw"
+    userns_mode: host
+    depends_on:
+      - chris
+    networks:
+      - local
+
+  db_migrate:
+    image: ghcr.io/fnndsc/cube:5.0.0
+    command:
+      - sh
+      - -c
+      - pip install -r /opt/ChRIS_ultron_backEnd/requirements/local.txt && python manage.py migrate --noinput
+    volumes:
+      - chris_files:/var/chris:rw
+      - ./chris_backend:/opt/app-root/src
```

Comment:
What is the reason we need to mount `/opt/app-root/src` and `/opt/ChRIS_ultron_backEnd`?

Also, should we use `:ro` on the source file mounts?

Reply:
Thanks for the feedback.

1. `/opt/app-root/src` is for the entrypoint, and `/opt/ChRIS_ultron_backEnd` is to easily know what we are running.
2. `:ro` is good to have, but the goal of this `docker-compose-dev.yml` is for development. I don't think `:ro` is necessary though～